client/src/component/Button/Button.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+import "./Button.scss";
+
+import { Props } from "./types";
+
+export const Button = ({ className, ...props }: Props): JSX.Element => {
+  return <button className={`${className} button`} {...props}></button>;
```

Comment:
1. 100% захочется задать кнопке какой-то текст. Нужно прокинуть соответствующий пропс

```suggestion
  return <button className={`${className} button`} {...props}>{text}</button>;
```

2. Предлагаю сразу подключить библиотеку [classnames](https://github.com/JedWatson/classnames) для более гибкой возможности управления классами

Reply:
1 исправил